packages/formstr-app/src/App.js

```diff
@@ -3,14 +3,23 @@ import "./App.css";
 import Routing from "./components/Routing";
 import { ProfileProvider } from "./provider/ProfileProvider";
 import { ApplicationProvider } from "./provider/ApplicationProvider";
+import { UserProvider } from "./provider/UserProvider";
+import { ListProvider } from "./provider/ListProvider";
+import { AppContextProvider } from "./provider/AppProvider";
 
 function App() {
   return (
     <HashRouter>
       <div className="App">
         <ProfileProvider>
           <ApplicationProvider>
-            <Routing />
+            <AppContextProvider>
```

Comment:
u see the redundancy in appcontextprovider when there is already an application provider in the line above this one?